LandLord-core/src/main/java/biz/princeps/landlord/multi/MultiClearInactiveTask.java

```diff
@@ -34,8 +35,13 @@ public MultiClearInactiveTask(ILandLord plugin, CommandSender commandSender, Col
 
     @Override
     public boolean process(OfflinePlayer offlinePlayer) {
-        clearedLands += wgManager.unclaim(
-                wgManager.getRegions(offlinePlayer.getUniqueId()));
+        LandClearInactiveEvent event = new LandClearInactiveEvent(commandSender, offlinePlayer);
+        Bukkit.getServer().getPluginManager().callEvent(event);
```

Comment:
I would in general avoid using the Bukkit god object and access the server via the plugin instance.

I just hope that paper will some day remove this class after the hard fork :D

Reply:
Ahah right, Bukkit "god" object is used through whole landlord, maybe we can replace most of them from now on?

Reply:
Yeah just try to avoid it from now an and just replace it whenever you see it and a plugin instance is already present